experiments/AtmosLES/squall_line.jl

```diff
@@ -122,8 +122,17 @@ end
   Read the original squall sounding
 """
 function read_sounding()
+
+    # Artifact creation is not thread-safe
+    #      https://github.com/JuliaLang/Pkg.jl/issues/1219
+    # To avoid race conditions from multiple jobs running this
+    # driver at the same time, we must store artifacts in a
+    # separate folder.
+
+    artifact_folder = mktempdir(@__DIR__; prefix = "artifacts_")
```

Comment:
Should note that if this is run with more than one MPI rank, each will download their own copy.

Reply:
Ah, right, makes sense. I'll update the first PR comment.

Reply:
Another solution is to
```julia
if MPI.Comm_rank(comm) == 0
    # download file
end
MPI.Barrier(comm)
```

Reply:
These are single-rank MPI jobs, so, IIUC, `mpirank = 1` for each separate job.

Reply:
ranks are 0-based (so there will always be a rank 0)

Reply:
Ah, but i see that the issues is across jobs, not within MPI jobs